src/components/ItaliaTheme/View/BandoView/BandoView.jsx

```diff
@@ -61,7 +74,11 @@ const BandoView = ({ content, location }) => {
           showbandostate={true}
         />
         <div className="row row-column-border border-light row-column-menu-left">
-          <aside className="col-lg-4">
+          <aside
+            className="col-lg-4"
+            role="navigation"
+            aria-label={intl.formatMessage(messages.sideMenuNavigation)}
```

Comment:
questi due attributi non era sufficiente metterli dentro al componente SideMenu?

Reply:
La region che viene identificata in automatico è dentro all'aside per questo viene identificata com l'aria-label, se lo mettiamo dentro al SideMenu crearà una terza regione anche se non mettiamo l'aria-label sull'aside.